### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by HarshnaNHaswani**
Please review the home page

### CODE REVIEW

**Comment by ashifkhn** [comment]
Looks good. Add some border radius in the main image(Next Update).

**Review by ashifkhn — APPROVED** [review]
Good To merge.

File: index-user.html [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+  <link rel="apple-touch-icon" sizes="180x180" href="/images/apple-touch-icon.png">
+  <link rel="manifest" href="/site.webmanifest">
+
+  <link rel="stylesheet" href="/styles/main.css">
+
+</head>
+
+<body class="Home">
+
+
```
Comment (manujagobind) [inline index-user.html:24]: nit: remove extra blank lines, there are many
Or use prettier

**Review by manujagobind — APPROVED** [review]

**Comment by HarshnaNHaswani** [comment]
Thanks! I'll do the necessary changes! @ashifkhn @manujagobind

**Pull request merged**